src/utils/deprecationWarning.js

```diff
@@ -12,3 +14,14 @@ export default function deprecationWarning(oldname, newname, link) {
     }
   }
 }
+
+export function collapsable(props, propName, componentName) {
+  if (props[propName] !== undefined) {
+    deprecationWarning(
+      `${propName} in ${componentName}`,
+      'collapsible',
+      'https://github.com/react-bootstrap/react-bootstrap/issues/425'
+    );
+  }
+  return React.PropTypes.bool.call(null, props, propName, componentName);
+}
```

Comment:
No problem. I was asking about it here https://github.com/react-bootstrap/react-bootstrap/issues/425#issuecomment-98238188, but it seems I'm asking a lot of questions in a row and some of them are lost :smile: